packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -420,6 +422,10 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
         }
       }
 
+      if (enableProfilerTimer) {
+        recordCommitTime();
```

Comment:
This is not the place for it. This is inside a hot loop and gets called thousands of time. This would be a better place: https://github.com/bvaughn/react/blob/7f027607b516d3cb3d58feeb07074813f6f8bfb4/packages/react-reconciler/src/ReactFiberScheduler.js#L632 That way we also ensure that the render time doesn't overlap with the commit time frame.

Reply:
An, whoops. Thanks~

Reply:
Ah, yeah. This was pretty dumb. Thanks for the pointer.